packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -1767,182 +1727,6 @@ export default class CBioPortalAPIInternal {
                 return response.body;
             });
         };
-    fetchClinicalDataViolinPlotsUsingPOSTURL(parameters: {
```

Comment:
taking this change out is going to cause type errors because this endpoint is being used. can you just do the genome nexus updates? the backend will be updated soon making these in sync